pkg/server/server.go

```diff
@@ -1603,8 +1604,15 @@ func (s *Server) PreStart(ctx context.Context) error {
 		}
 	})
 
+	// After setting modeOperational, we can block until all stores are fully
+	// bootstrapped.
 	s.grpc.setMode(modeOperational)
 
+	// Block until all stores are fully bootstrapped.
+	log.Infof(ctx, "waiting for all stores to be fully bootstrapped")
+	waitForBootstrapNewStores()
+	log.Infof(ctx, "all stores fully bootstrapped")
```

Comment:
Let's remove this logging statement, it's not necessary given we're logging right after so can infer that everything has been fully bootstrapped.